.github/workflows/ci.yml

```
@@ -2,91 +1,147 @@
+      
+      - name: Install dependencies
         run: |
-          pnpm db:generate
-          pnpm db:push
+          pnpm install --frozen-lockfile
+      
+      - name: Build Tauri app
+        run: pnpm tauri build
+        env:
+          TARGET_ARCH: x86_64
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)